combat-trainer.lic

```diff
@@ -1425,6 +1425,13 @@ class SpellProcess
     true
   end
 
+  def cast_runestone(data, game_state)
+    prepare_to_cast_runestone(data, @settings)
+    return unless DRCI.in_hands?(data['runestone_name'])
+    prepare_spell(data, game_state, @buff_force_cambrinth)
+    DRCI.put_away_item?(data['runestone_name'], @settings.runestone_storage)
```

Comment:
If the rune is disposed of when trying to prepare it then your hand is now empty and this will try to stow nothing. To avoid that step we can check if the hands are still holding the rune.
```suggestion
    return unless DRCI.in_hands?(data['runestone_name'])
    DRCI.put_away_item?(data['runestone_name'], @settings.runestone_storage)
```